### PR TITLE
Add buildSrc, and utility functions

### PR DESCRIPTION
**Pull request by nomisRev**
Removes `credentials.gradle` and replaces it with buildSrc, and Gradle properties.
It exposes the variables as system variables in test enviroment.

### CODE REVIEW

**Review by timrijckaert — APPROVED** [review]

File: build.gradle.kts [hunk 1]
```diff
@@ -27,18 +23,14 @@ allprojects {
             targetCompatibility = JavaVersion.VERSION_11.toString()
         }
 
+        val username by variable("username")
```
Comment (timrijckaert) [inline build.gradle.kts:26]: Shouldn't this also be `goplay.username`?

**Comment by timrijckaert** [comment]
Do we need to update the `DEVELOPMENT.md` for these changes?

File: build.gradle.kts (hunk 1 above)
Comment (nomisRev) [reply build.gradle.kts:26]: What is in `local.properties`? I thought enventually we'd have:

```kotlin
val username by variable("username")

listOf("goplay", "vtm").forEach { brand ->
  enviroment("${brand}.username, username)
}
```

Or should it rather be:

```kotlin
listOf("goplay", "vtm").forEach { brand ->
  val username by variable("${brand}.username")
  enviroment("${brand}.username, username)
}
```

**Comment by nomisRev** [comment]
Yes, we should update `DEVELOPMENT.md`.
I believe this solution is more flexible than what we had before.

File: build.gradle.kts (hunk 1 above)
Comment (nomisRev) [reply build.gradle.kts:26]: If the later, then yes we should change it to `goplay.username`

Comment (timrijckaert) [reply build.gradle.kts:26]: We had following structure. 

```
vrtnu.username=
vrtnu.password=

vtmgo.username=
vtmgo.password=

goplay.username=
goplay.password=
```

Unless we have 1 dummy account where all username and passwords are equal for all services.

Comment (nomisRev) [reply build.gradle.kts:26]: Alright, makes sense. Let me update the code.

**Pull request merged**